opal/mca/patcher/patcher.h

```diff
@@ -30,9 +30,9 @@
  * Reference: "64-bit PowerPC ELF Application Binary Interface Supplement 1.9" */
 #    define OPAL_PATCHER_BEGIN                      \
         unsigned long toc_save;                     \
-        asm volatile("std 2, %0" : "=m"(toc_save)); \
-        asm volatile("nop; nop; nop; nop; nop");
-#    define OPAL_PATCHER_END asm volatile("ld  2, %0" : : "m"(toc_save));
+        asm volatile("std 2, %0" : "=m"(toc_save) :: "memory"); \
```

Comment:
Should we add a reference to the GCC ticket to remember why we globber the memory here? Just in case someone looks at this in 3 years from now and tries to save some cycles...

Reply:
@bosilca ^

Reply:
dang, I wanted to add this link to the commit. Sorry.